src/library/dlgtrackinfo.cpp

```diff
@@ -377,6 +374,7 @@ void DlgTrackInfo::saveTrack() {
 
     m_pLoadedTrack->setKeys(m_keysClone);
 
+    bool theCueFound = false;
```

Comment:
"theCueFound" -> "loadCueFound"? We have a name for "the" cue.

Reply:
There is no "load cue". The main cue can be used that way with the option to load decks there in the Deck preferences, but it is misleading to call it a "load cue" in code.

Reply:
But this flag tracks if the LOAD cue has been set during the iteration. This purpose should be reflected in the naming.

Reply:
The misnamed "LOAD" cue will be renamed to MainCue in #2103.

Reply:
.. and this is a open discussion without a final conclusion: 
https://github.com/mixxxdj/mixxx/pull/2103#discussion_r292439905

Reply:
I'm just asking for a consistent naming, choose whatever you want.

Reply:
I am unsure how to continue here. I would prefer "The Cue" here and in #2103, because this is the name we have used since introducing hot cues. But I can also live with MainCue, if we all agree to this name. 
I can use here loadCueFound and it can be changed in #2103 to the new name or introduce it here. Wat do you think?

Reply:
Just leave it as is. I didn't plan to start a fruitless discussion. It was just a minor remark.